dags/openshift_nightlies/manifest.yaml

```diff
@@ -1,8 +1,4 @@
 versions:
-- version: "4.10"
```

Comment:
this should not be removed, instead previous to be `4.11`, stable to `4.12` and next to `4.13` but Managed Service cluster has nothing to do with these `versions` so I'd leave this to self-managed team handle this rollout.

Reply:
@krishvoor lets keep this and remove 4.10 from others too